### PR TITLE
 Configure TLS for rsyslog remote logging. 

### PR DESCRIPTION
**Pull request by adelton**
#### Description:

- Configure TLS for rsyslog remote logging. 

#### Rationale:

- For protection of data being logged, the connection to the remote logging server needs to be authenticated and encrypted.

### CODE REVIEW

File: linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/rule.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Configure TLS for rsyslog remote logging'
+
+description: |-
+    Configure <tt>rsyslog</tt> to use Transport Layer
+    Security (TLS) support for logging to remote server.
```
Comment (jan-cerny) [inline linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/rule.yml]: We usually describe also which option needs to be configured and in which configuration file this option can be found.

Comment (adelton) [reply linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/rule.yml]: Rule descriptions expanded -> c49f0b5b32ef05db833004a77a8183e065e1793b.

File: linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/oval/shared.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <ind:textfilecontent54_test id="test_rsyslog_remote_tls"
+    comment="tests the omfwd action configuration"
+    check_existence="at_least_one_exists" check="all" version="1">
+    <ind:object object_ref="obj_rsyslog_remote_tls" />
+    <ind:state state_ref="state_rsyslog_remote_tls" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="obj_rsyslog_remote_tls" version="1">
+    <ind:filepath operation="pattern match">^/etc/rsyslog\.(conf|d/.+\.conf)$</ind:filepath>
+    <ind:pattern operation="pattern match">^\s*action\(type="omfwd"(.+?)\)</ind:pattern>
```
Comment (jan-cerny) [inline linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/oval/shared.xml]: @adelton This works only if the whole command is on a single line. From a quick look into rsyslog conf, it seems it's possible to have newlines and other whitespace characters inside the parentheses. Could you take a look at it?

Comment (jan-cerny) [reply linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/oval/shared.xml]: you can try `<behaviors singleline="true"/>`

Comment (adelton) [reply linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/oval/shared.xml]: Added as 843c7f1978ed43324fa621888afdc2563b06e124.

**Pull request merged**